### PR TITLE
Report page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
I have added a Report Page,
-It contains two Section, first one store the total Score, Correct Answer, Wrong Answer, and Attempt of all the Category, and the Second one store the total Score, Correct Answer, Wrong Answer, and Attempt of the category.
Screenshot (333)

### CODE REVIEW

**Review by nikhilmalviya-dev — APPROVED** [review]
code is clean.. you can merge

File: src/App.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Home } from "../src/Pages/Home-Page/Home";
 // import { Signup } from "../src/Pages/Signup-Page/Signup";
```
Comment (ShreyashMahajan) [inline src/App.js:4]: you can remove the commented code

File: src/Context/QuizContext.jsx [hunk 2]
```diff
@@ -5,7 +5,52 @@ const QuizContext = createContext();
 const QuizProvider = ({ children }) => {
   const [state, dispatch] = useReducer(QuizReducer, {
     CurrentQuiz: {},
-    Result: { score: 0, correct: 0, wrong: 0, attempt: 0 },
+    Result: { score: 0, correct_answer: 0, wrong_answer: 0, attempt: 0 },
+    Category_Report: [
+      {
+        category: "History",
+        score: 0,
```
Comment (ShreyashMahajan) [inline src/Context/QuizContext.jsx:12]: i liked how you create different section for storing marks of different categories

File: src/Context/QuizContext.jsx [hunk 3]
```diff
@@ -5,7 +5,52 @@ const QuizContext = createContext();
 const QuizProvider = ({ children }) => {
   const [state, dispatch] = useReducer(QuizReducer, {
```
Comment (ShreyashMahajan) [inline src/Context/QuizContext.jsx:6]: more specific name can be used here for state and dispatch

File: src/Pages/Home-Page/home.css [hunk 4]
```diff
@@ -1,18 +1,14 @@
 .home-page {
   font-family: "Montserrat", sans-serif;
   display: grid;
-  grid-template-rows: 4rem auto 5rem;
-  grid-template-columns: 250px auto;
-
+  grid-template-rows: 4rem minmax(calc(100vh - 9rem), 1fr) 5rem;
   grid-template-areas:
```
Comment (ShreyashMahajan) [inline src/Pages/Home-Page/home.css:5]: only 'header'
        main
       'footer'
can be used here

**Review by ShreyashMahajan — APPROVED** [review]
Overall code is very understandable, neat, and clean. Just look at the suggestion. Then good to merge.

**Comment by SaurabhChawda** [comment]
> code is clean.. you can merge

ThankYou

File: src/Context/QuizContext.jsx (hunk 3 above)
Comment (SaurabhChawda) [reply src/Context/QuizContext.jsx:6]: Okay, I Will change.

**Pull request merged**